src/v/kafka/server/handlers/topics/topic_utils.cc

```diff
@@ -31,11 +34,11 @@ void append_cluster_results(
       });
 }
 
-ss::future<std::vector<model::node_id>> wait_for_leaders(
+ss::future<> wait_for_leaders(
   cluster::metadata_cache& md_cache,
   std::vector<cluster::topic_result> results,
   model::timeout_clock::time_point timeout) {
-    std::vector<ss::future<model::node_id>> futures;
+    ss::chunked_fifo<ss::future<>> futures;
```

Comment:
can't this be removed? It doesn't look to be used.

Reply:
exactly it should be removed